apps/admin/backend/src/app.ts

```diff
@@ -898,6 +905,35 @@ function buildApi({
       });
     },
 
+    async printWriteInAdjudicationReport(): Promise<void> {
+      return printWriteInAdjudicationReport({
+        store,
+        printer,
+        logger,
+        userRole: assertDefined(await getUserRole()),
```

Comment:
having to pass the `userRole` around in order to pass it to the logger is getting a little silly. not for this PR, but we ought to refactor this to logger perhaps has that state, or a hook into it. @jonahkagan has brought this up before